<launch>
    <param name="init_pose_x" value="0.0" />
    <param name="init_pose_y" value="0.0" />
    <param name="init_pose_yaw" value="0.0" />
    
    <param name="duration_clear_costmap" value="3" />
    <param name="base_frame" value="base_footprint" />
    <param name="map_frame" value="map" />
    <param name="laser_frame" value="armbot_laser_scanner" />
    <param name="odom_frame" value="odom" />

    <param name="map_topic" value="map" />
    <param name="laser_topic" value="scan" />
    <param name="odom_topic" value="odom" />
    <param name="initial_pose_topic" value="initialpose" />
    <param name="filtered_scan_topic" value="filtered_scan_points" />
    <param name="move_base_goal_topic" value="move_base_simple/goal" />
    <param name="move_base_action_name" value="move_base" />
    <param name="move_base_clear_costmaps_service_name" value="move_base/clear_costmaps" />

    <param name="use_full_search" value="false" />
    <param name="radius_near_obstacles" value="2" />
    <param name="static_obstacle_threshold" value="5" />
    <param name="deg_rotation" value="4" />
    <param name="lidar_score_threshold" value="0.7" />

    <node name="lidar_loc_node" pkg="lidar_loc" type="lidar_loc_node" output="screen" />
</launch>
